specification/cognitiveservices/data-plane/AnomalyDetector/preview/v2.0-preview/RootCauseAnalysis.json

```
@@ -0,0 +1,2143 @@
+    "version": "2.0-preview"
+  },
+  "paths": {
+    "/rootCauseAnalysis/dataSets": {
+      "put": {
+        "tags": [
+          "RootCauseAnalysisDataSet"
+        ],
+        "summary": "Create a root cause analysis data set",
+        "operationId": "CreateRootCauseAnalysisDataSet",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "create a root cause analysis data set request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RootCauseAnalysisDataSet"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success",
+            "headers": {
+              "Location": {
+                "description": "Location of the newly created resource.",
+                "type": "string"
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "enum": [
+                "DisplayNameAlreadyExist",
+                "InvalidGranularityNone",
+                "SizeLimitExceeded",
+                "InvalidRequestSchema"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "MultiDimRCACreateDataSetErrorCode",
+                "modelAsString": true
+              }
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "String identical to error code.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a root cause analysis data set": {
+            "$ref": "./examples/CreateRootCauseAnalysisDataSet.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "RootCauseAnalysisDataSet"
+        ],
+        "summary": "List root cause analysis data set",
+        "operationId": "ListRootCauseAnalysisDataSet",
```

I think this is listing all the data sets, right?
```suggestion
        "summary": "List root cause analysis data sets",
        "operationId": "ListRootCauseAnalysisDataSets",
```

---

updated in new PR